src/loader.js

```diff
@@ -1,6 +1,7 @@
-const path = require('path');
 const { Template } = require('webpack');
 const { refreshUtils } = require('./runtime/globals');
+const RefreshModuleRuntime = require('./runtime/RefreshModuleRuntime');
+const RefreshModuleRuntimeString = Template.getFunctionContent(RefreshModuleRuntime);
```

Comment:
I thought there was already a comment here about this, but if this line is safe to remove from the function, is there any reason the `trim()` or the `replace` calls can't also be brought out and only run once?